README.md

```diff
@@ -35,10 +35,6 @@ cd ..
 # Build the images
 make all
 docker compose -f docker-compose-build.yaml build
-
-# Alternatively you can build specific NF image e.g.:
-make amf
```

Comment:
@ianchen0119 , I think it is ok to re-add back this to README.md; I tested this PR branch with building a specific NF with customized code (smf), then replaced `image:` in its docker-compose.yaml entry with the built one (`free5gc/smf:latest)` and it worked.